interfaces/compiler/src/main/java/compiler/interfaces/Logger.java

```diff
@@ -0,0 +1,7 @@
+package compiler.interfaces;
+
+import xsbti.Severity;
```

Comment:
You should not depend on `xsbti` at all. I suggest you add 3 methods `info`, `warn` and `error`.

Reply:
ok